cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseTableTest.java

```diff
@@ -357,6 +368,80 @@ public void testTableWithPermissions() throws IOException {
     Assert.assertFalse(admin.exists());
   }
 
+  @Test
+  public void testScannerCache() throws Exception {
+    String tableName = "scanCache";
+    // note: it appears that HBase only enforces the scanner timeout after 10 seconds.
+    // setting it to 3 seconds does not mean it will actually fsail after 3 sweconds.
+    // therefore we have to cross the 10 seconds. here: 1200 times 10ms sleep.
+    int numRows = 1200;
+    DatasetAdmin admin = getTableAdmin(CONTEXT1, tableName);
```

Comment:
I tried creating a table with the property `CONFIG_HBASE_CLIENT_CACHE_BLOCKS`, but it didn't seem to take effect due to how `HBaseTableAdmin#create` is implemented.
Can you try that out? Related to https://github.com/caskdata/cdap/pull/9133#discussion_r123405609.